tests/PayloadSizeTest.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * GPII-1880 Test Definitions
+ *
+ * Copyright 2017 Raising The Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    jqUnit = fluid.require("node-jqunit", require, "jqUnit"),
+    gpii = fluid.registerNamespace("gpii"),
+    kettle = fluid.registerNamespace("kettle");
+
+require("../index.js");
```

Comment:
We should generally standardise on fluid.require("%universal") as a bit more meaningful and stable